src/fastoad/openmdao/utils.py

```diff
@@ -92,11 +92,20 @@ def get_problem_after_setup(problem: om.Problem) -> om.Problem:
     having the problem being actually setup.
 
     :param problem:
-    :return: the problem itself it setup() has already been run, or a copy of the provided problem after setup()
-             has been run
+    :return: the problem itself it setup() has already been run, or a copy of the provided problem
```

Comment:
Typo "it" au lieu de "if"

Reply:
Yep. Corrigé en 199623e69b5bac1cfd6b1815c1a58dae26831fe9